src/ndb_p1_section.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+
+#include "ndb_p1_section.h"
+
+#include <string.h>
+#include "ndb.h"
+
+#define GPS_L1_EPHE_FILE_NAME "gps_l1_ephe"
```

Comment:
I think in general we should combine where possible. Although specifically for almanac and ephemeris, those are probably one per physical satellite and not one per signal. Maybe I should add an `sid_to_satellite_index()` API or something.